src/com/sap/piper/integration/TransportManagementService.groovy

```diff
@@ -131,13 +141,7 @@ class TransportManagementService implements Serializable {
             validResponseCodes    : "100:399"
```

Comment:
My feeling is that the consoleLogResponseBody parameter above should not depend on config.verbose setting and should be either set to false (I think, such a change was already provided in a different commit) or simply omitted, if false is a default value.

Reply:
Again: you are claiming about some code which has not been changed by the PR. IMO it is important to keep PRs small and focused. In case there is something which can be improved, but it not related to the PR why not opening another PR?

>My feeling is that the consoleLogResponseBody [...]

See PR  #1301 which has been merged in the meantime.

> or simply omitted, if false is a default value.

having it explicitly provided with value `false` together with a commit explaining why it better IMO than simply omitting it, even if it is the default

Reply:
> having it explicitly provided with value `false` together with a commit explaining why it better IMO than simply omitting it, even if it is the default

Makes sense.